### ISSUE TITLE
Blender-Cats Plugin Add On Error 'pose' object has no attribute 'bone_group'

### ISSUE THREAD

**Issue by NiteSek** [absolute-quantum/cats-blender-plugin#674]
Can't figure this out Added the Cats Plugin/Add-on to Blender & attempted to use the "Fix Model" button within Blender & then was immediately presented with this error 👇

Python: Traceback (most recent call last):
File "C:\Users\nites\AppData\Roaming\Blender Foundation\Blender\4.1\scripts\addons\cats-blender-plugin-0-19-0\tools\armature.py", line 468, in execute for group in armature.pose.bones_groups:
                     ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Pose' object has no attribute 'bone_groups'

If someone can PLEEEEEEASE help me figure out how to fix this, then pls let me know ASAP. I'm dying over here Lol

**Comment by RinaDev-xyz** [comment]
Please read this first

https://github.com/absolute-quantum/cats-blender-plugin/issues/670

**Comment by bunbuntsuki** [comment]
Has anyone figured this issue out? I'm having the same problem

**Comment by RinaDev-xyz** [comment]
The answer lies in my last post..... 

Aka maybe use unofficial and not the broken and abandoned official cats.... https://github.com/absolute-quantum/cats-blender-plugin/issues/693

**Comment by bunbuntsuki** [comment]
Yeah except I've tried the dev cat plug and that doesn't work either says TESTING is missing but I've checked the scripts and they're there. Honestly completely gave up for today and deleted my project. Figured it would be best to just undownload Blender and re-download the older version since I can't get anything to work so far. And nothing else has been as detailed or helpful so.

**Comment by RinaDev-xyz** [comment]
Or you could use unofficial Cats like i just said and have all the bugs fixed...

**Comment by bunbuntsuki** [comment]
Right but in the description is literally says that for 4.3 it may or may not work and there's lots of bugs?? I mean I'll try it tomorrow but it doesn't look like it'll make it easier. But we'll see.

**Comment by RinaDev-xyz** [comment]
The post is outdated, I not constantly updating it, you need to check the GitHub or the website to see the latest information.

Also where does it say that as I don't see a single mention of 4.3 in my post.

If you went to the 4.3 branch that readme isn't updated, all stable releases are in the releases tab, all the latest information is in the welcome branch readme. You shouldn't even have to go to the 4.3 branch.

The only unstable unofficial cats right now is for Blender 4.4 which is in Alpha.

4.3 is fully supported and has been for a week now.

**Comment by z4m1** [comment]
armature.pose.bones_groups # spell error
armature.pose.bone_groups # 4.x error

armature.data.collections # correct
